assets/styles.css

```diff
@@ -0,0 +1,814 @@
+  margin-left: 10px;
+  height: 7rem;
+  width: 15rem;
+  opacity: 0.4;
+  z-index: -1;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-position-x: left;
+  background-size: contain;
+}
+
+.tnw-red {
+  width: 100%;
+  height: 70%;
+  opacity: 0.75;
+}
+
+.second-background-img {
+  background-image: url("./gradient-img.jpg");
+  height: 49%;
+  opacity: 1;
+  z-index: 2;
+  background-position: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.third-background-img {
+  background-image: url("./third-background-img.jpg");
+  height: 49%;
+  opacity: 1;
+  z-index: 2;
+  background-position: center;
+  margin-top: 2%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+.tnw-down {
+  display: flex;
+  opacity: 1;
+}
+
+footer p {
+  color: white;
+  font-size: 0.8em;
+  margin-top: 17px;
+}
+
+.tnw-down p {
+  color: white;
+  opacity: 1;
+}
+
+.tnw-down i {
+  padding-top: 18px;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.tnw-container {
+  margin-bottom: 0;
+  height: 35%;
+}
+
+.third-background-img div {
+  display: flex;
+}
+
+.second-background-img p,
+.third-background-img p {
+  margin-block-start: 0;
+}
+
+.four-grid-container {
+  display: grid;
+  grid-gap: 20px;
+  grid-template-columns: repeat(4, 1fr);
+  max-width: 70%;
+  margin: 0 auto;
+  box-sizing: border-box;
+  justify-content: center;
+}
+
+.four-grid-container h2 {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  color: red;
+}
+
+.second-headings {
+  display: flex;
+  grid-column-start: 1;
+  grid-column-end: 5;
+  color: red;
+  justify-content: space-between;
+  margin-top: 25px;
+}
+
+.grid-img img {
+  width: 100%;
+  height: 100%;
+}
+
+.grid-img p {
+  font-size: 0.8em;
+  color: #aaa;
+}
+
+.non-flex-section3 > h2 {
+  padding-left: 27%;
+  color: red;
+}
+
+.small-background-img h2 {
+  color: red;
+}
+
+.second-background-img *,
+.third-background-img * {
+  color: white;
+  text-decoration: none;
+}
+
+.second-headings h1 {
+  margin-right: 15px;
+  font-size: 1.3em;
+}
+
+.s-image {
+  height: 50px;
+  width: 50px;
+  position: relative;
+  top: 75px;
+  margin-left: 10px;
+}
+
+.non-flex-section {
+  display: block;
+}
+
+.spiff-background {
+  background-color: red;
+  color: white;
+}
+
+.venture-background {
+  background-color: #ffece7;
+}
+
+.non-flex-section .spiff-background,
+.non-flex-section .venture-background {
+  padding-left: 22%;
+  margin-right: 25px;
+}
+
+.spiff-background h4 {
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-bottom: 20px;
+  padding-top: 10px;
+  margin-left: 70px;
+}
+
+.venture-background p {
+  margin-top: 0;
+  margin-bottom: 10px;
+  margin-left: 70px;
+}
+
+.second-flex-div i {
+  margin-block-start: 1em;
+  margin-right: 10px;
+}
+
+.p-tag {
+  border-top: 1px solid pink;
+  font-size: 0.7em;
+}
+
+.flex {
+  display: flex;
+}
```

Comment:
Same style definition of the selector on line 289. Group selectors, or create a new class.